### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by zhmushan**

### CODE REVIEW

**Comment by zhmushan** [comment]
I can do nothing about it until I find a Windows device.

**Comment by axetroy** [comment]
Maybe we can upgrade one by one. This is also the easiest way to upgrade deps.

**Comment by axetroy** [comment]
截屏2020-07-31 19 37 51

I found that the reason for CI failure is prettier.

Maybe we can skip this step in Windows CI.

**Comment by zhmushan** [comment]
I skipped the format check on windows...

File: package-lock.json [hunk 1]
```diff
@@ -5261,189 +5761,181 @@
...
+      "resolved": "http://registry.npm.qima-inc.com/wrappy/download/wrappy-1.0.2.tgz",
       "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=",
       "dev": true
     },
     "write-file-atomic": {
-      "version": "2.4.1",
-      "resolved": "https://registry.npmjs.org/write-file-atomic/-/write-file-atomic-2.4.1.tgz",
-      "integrity": "sha512-TGHFeZEZMnv+gBFRfjAcxL5bPHrsGKtnb4qsFAws7/vlh+QfwAaySIw4AXP9ZskTTh5GWu3FLuJhsWVdiJPGvg==",
+      "version": "3.0.3",
+      "resolved": "http://registry.npm.qima-inc.com/write-file-atomic/download/write-file-atomic-3.0.3.tgz",
```
Comment (axetroy) [inline package-lock.json:5876]: Can you use npm or yarn registry?

Comment (zhmushan) [reply package-lock.json]: Thanks for pointing out😥

**Comment by axetroy** [comment]
We should run `npm run release` to update `dist/index.js`

> BTW. The command `npm run release` is ambiguous. it makes people confuse.

**Comment by zhmushan** [comment]
Could we merge this first? I will continue to make some changes tomorrow.

**Pull request merged**